### DISCUSSION TITLE
Judgement editor

### DISCUSSION THREAD

**Discussion by Senhorcha2** [ppy/osu#24667] [Ideas] [answered]
Add an option for players to be able to change the judgment like quaver, change the ms of miss PEEFECT etc.

**Comment by peppy** [comment, marked as answer]
You can do this using the "Difficulty Adjust" mod (change "Overall Difficulty"). We won't be adding any further adjustments beyond what this offers as it's likely not productive.

**Comment by Senhorcha2** [comment]
Adjusting difficulty as you speak is not O? because if that's the case,
there's no way you can't see ms das judge

Em seg., 28 de ago. de 2023 01:01, Dean Herbert ***@***.***>
escreveu:

**Comment by Givikap120** [comment]
> Add an option for players to be able to change the judgment like quaver, change the ms of miss PEEFECT etc.

tbh it feels unnecessary because everyone cares only about 300-100 boundary position what easily controllable via OD
nobody cares about 100-50 boundary and 50-miss boundary
